{
  "document_title": "CVE-2020-28975",
  "document_type": "SUSE CVE",
  "document_publisher": {
    "type": "Vendor",
    "contact_details": "security@suse.de",
    "issuing_authority": "SUSE Security Team"
  },
  "document_tracking": {
    "identification": {
      "id": "SUSE CVE-2020-28975"
    },
    "status": "Interim",
    "version": "1",
    "revision_history": {
      "revision": {
        "number": "6",
        "date": "2022-11-27T00:53:11Z",
        "description": "current"
      }
    },
    "initial_release_date": "2021-05-30T14:45:28Z",
    "current_release_date": "2022-11-27T00:53:11Z",
    "generator": {
      "engine": "cve-database/bin/generate-cvrf-cve.pl",
      "date": "2020-12-27T01:00:00Z"
    }
  },
  "document_notes": [
    {
      "text": "CVE-2020-28975",
      "title": "CVE",
      "type": "Summary"
    },
    {
      "text": "** DISPUTED ** svm_predict_values in svm.cpp in Libsvm v324, as used in scikit-learn 0.23.2 and other products, allows attackers to cause a denial of service (segmentation fault) via a crafted model SVM (introduced via pickle, json, or any other model permanence standard) with a large value in the _n_support array. NOTE: the scikit-learn vendor's position is that the behavior can only occur if the library's API is violated by an application that changes a private attribute.",
      "title": "Mitre CVE Description",
      "type": "Description"
    },
    {
      "text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
      "title": "Terms of Use",
      "type": "Legal Disclaimer"
    }
  ],
  "document_references": [
    {
      "type": "Self",
      "url": "https://www.suse.com/support/security/rating/",
      "description": "SUSE Security Ratings"
    }
  ],
  "product_tree": {
    "branch": [
      {
        "type": "Product Family",
        "name": "openSUSE Tumbleweed",
        "branch": [
          {
            "type": "Product Name",
            "name": "openSUSE Tumbleweed",
            "full_product_name": {
              "text": "openSUSE Tumbleweed",
              "product_id": "openSUSE Tumbleweed",
              "cpe": "cpe:/o:opensuse:tumbleweed"
            }
          }
        ],
        "full_product_name": {}
      },
      {
        "type": "Product Version",
        "name": "python310-scikit-learn-1.0.2-1.1",
        "full_product_name": {
          "text": "python310-scikit-learn-1.0.2-1.1",
          "product_id": "python310-scikit-learn-1.0.2-1.1"
        }
      },
      {
        "type": "Product Version",
        "name": "python38-scikit-learn-1.0.2-1.1",
        "full_product_name": {
          "text": "python38-scikit-learn-1.0.2-1.1",
          "product_id": "python38-scikit-learn-1.0.2-1.1"
        }
      },
      {
        "type": "Product Version",
        "name": "python39-scikit-learn-1.0.2-1.1",
        "full_product_name": {
          "text": "python39-scikit-learn-1.0.2-1.1",
          "product_id": "python39-scikit-learn-1.0.2-1.1"
        }
      }
    ],
    "relationship": [
      {
        "product_reference": "python310-scikit-learn-1.0.2-1.1",
        "relation_type": "Default Component Of",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      },
      {
        "product_reference": "python38-scikit-learn-1.0.2-1.1",
        "relation_type": "Default Component Of",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      },
      {
        "product_reference": "python39-scikit-learn-1.0.2-1.1",
        "relation_type": "Default Component Of",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      }
    ]
  },
  "vulnerability": [
    {
      "notes": [
        {
          "text": "** DISPUTED ** svm_predict_values in svm.cpp in Libsvm v324, as used in scikit-learn 0.23.2 and other products, allows attackers to cause a denial of service (segmentation fault) via a crafted model SVM (introduced via pickle, json, or any other model permanence standard) with a large value in the _n_support array. NOTE: the scikit-learn vendor's position is that the behavior can only occur if the library's API is violated by an application that changes a private attribute.",
          "title": "Vulnerability Description",
          "type": "General"
        }
      ],
      "cve": "CVE-2020-28975",
      "product_statuses": [
        {
          "type": "Fixed",
          "product_id": [
            "openSUSE Tumbleweed:python310-scikit-learn-1.0.2-1.1",
            "openSUSE Tumbleweed:python38-scikit-learn-1.0.2-1.1",
            "openSUSE Tumbleweed:python39-scikit-learn-1.0.2-1.1"
          ]
        }
      ],
      "threats": [
        {
          "type": "Impact",
          "description": "important"
        }
      ],
      "cvss_score_sets": {
        "score_set_v_2": {
          "base_score_v_2": "5",
          "vector_v_2": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
        },
        "score_set_v_3": {
          "base_score_v_3": "7.5",
          "vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
        }
      }
    }
  ]
}
